cmd/querier/main.go

```diff
@@ -88,6 +88,7 @@ func main() {
 
 	subrouter := server.HTTP.PathPrefix("/api/prom").Subrouter()
 	subrouter.PathPrefix("/api/v1").Handler(middleware.AuthenticateUser.Wrap(promRouter))
+	subrouter.Path("/remote_read").Handler(middleware.AuthenticateUser.Wrap(http.HandlerFunc(queryable.Q.RemoteReadHandler)))
```

Comment:
Though maybe `remote_read` is better, since that's what we officially call it in Prometheus (in the config and such).